lib/fluent/plugin/in_s3.rb

```diff
@@ -275,6 +279,10 @@ def create_sqs_client
       options[:region] = @s3_region if @s3_region
       options[:endpoint] = @sqs.endpoint if @sqs.endpoint
       options[:http_proxy] = @proxy_uri if @proxy_uri
+      if @sqs.aws_key_id && @sqs.aws_sec_key
```

Comment:
Could you raise an error at `configure` when one of them is lacked so that users can noticed their mistake?
https://github.com/fluent/fluent-plugin-s3/blob/fba5e07d1eb7908eb3d6c42abed161e7b8decba2/lib/fluent/plugin/in_s3.rb#L139-L141